src/consumer/base_consumer.rs

```diff
@@ -235,6 +235,15 @@ impl<C: ConsumerContext> Consumer<C> for BaseConsumer<C> {
         }
     }
 
+    fn store_offset_list(&self, tpl: &TopicPartitionList) -> KafkaResult<()> {
```

Comment:
In an effort to stay closer to `librdkafka`'s naming schema, would it make sense to name the functions `store_offsets`?